### PR TITLE
Added missing error messages

### PR DESCRIPTION
**Pull request by SohailHaqyar**

### CODE REVIEW

File: src/error-details/gateway.ts [hunk 1]
```diff
@@ -81,4 +81,41 @@ export const gatewayErrorDetailsMap: IDetailsMap = {
     technichalReason:
       'The proxyService.getLogs function threw an error when attempting to fetch proxy logs',
   },
+
+  [Details.RemoteConnectInfoByIdNotFoundInDb]: {
+    friendlyMessage:
+      'A critical server error has occured. Please contact support.',
+    technichalReason:
+      'The remoteConnectService.findById returned a falsy value.',
```
Comment (TonyH-Salient) [inline src/error-details/gateway.ts:89]: NIT: `remoteConnectService.findById()`

File: src/error-details/organization.ts [hunk 2]
```diff
@@ -114,4 +114,17 @@ export const organizationDetailsMap: IDetailsMap = {
     technichalReason:
       'An organization registration is already in progress with this name',
   },
+
+  [Details.OrganizationByIdNotFoundInDb]: {
+    friendlyMessage:
+      'The action requires you to be a member of an organization. Either create an organization or join one to enable this action.',
+    technichalReason:
+      'organizationService.findById did not return a truthy value',
```
Comment (TonyH-Salient) [inline src/error-details/organization.ts:122]: Same here `organizationService.findById()`

**Review by TonyH-Salient — APPROVED** [review]

File: src/error-details/organization.ts (hunk 2 above)
Comment (SohailHaqyar) [reply src/error-details/organization.ts]: [fixed super nits](https://github.com/SalientSys/salient-sst-errors/pull/17/commits/bc0734a9ac8d0b6afcc6701318f0b6d746ce40fc)

File: src/error-details/gateway.ts (hunk 1 above)
Comment (SohailHaqyar) [reply src/error-details/gateway.ts]: [fixed super nits](https://github.com/SalientSys/salient-sst-errors/pull/17/commits/bc0734a9ac8d0b6afcc6701318f0b6d746ce40fc)

**Pull request merged**